lib/internal/util.js

```diff
@@ -654,7 +654,25 @@ function isArrayBufferDetached(value) {
   return false;
 }
 
+/**
+ * @template T return value of loader
```

Comment:
What is a “loader” in this context?

Reply:
Added some comments about the parameter.